admiral/pkg/controller/common/identity.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+import "istio.io/istio/pkg/log"
+
+var workloadIdentifier = Identity
```

Comment:
I am assuming the intention here is to manage the overriden Identity instead of passing around AdmiralParams? If so, can we have params here instead of just `workloadIdentifier`. That way we can consume any overriden parameters via this file/param.